### PR TITLE
Updated readme...

### PR DESCRIPTION
**Pull request by rilango**
Apart from updated readme, this comming also include a change to add a verb for generating cache in lauch.sh. This is essentially a wrapper for the same verb in ./startdash.py file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,32 +2,36 @@
...
 
-To customize your local environment, edit the appropriate section of `launch.sh` or provide a `~/.cheminf_local_environment` file with the following information below.  
-To generate a template for `.cheminf_local_environment`, just run `./launch.sh` with no arguments.  
+To customize your local environment, edit the appropriate section of `launch.sh` or provide a `~/.cheminf_local_environment` file with the following information below.
+To generate a template for `.cheminf_local_environment`, just run `./launch.sh` with no arguments.
 If `.cheminf_local_environment` does not exist, then a template will be written for you.
 
 ```
-CONT=gitlab-master.nvidia.com:5005/<path_to_your_repo>
+CONT=gitlab-master.nvidia.com:5005/cheminf-clustering
```
Comment (mlgill) [inline README.md:25]: Should this (and the default in the launch.sh script) be changed to what is appropriate for NGC, where the container is being hosted? Users won't have access to this.

File: README.md [hunk 2]
```diff
@@ -2,32 +2,36 @@
...
 REGISTRY=gitlab-master.nvidia.com:5005
```
Comment (mlgill) [inline README.md:26]: Same here.

File: README.md [hunk 3]
```diff
@@ -2,32 +2,36 @@
...
 REGISTRY_USER='$oauthtoken'
 REGISTRY_ACCESS_TOKEN=$(cat <path_to_your_access_token>)
```
Comment (mlgill) [inline README.md:28]: This is an NGC registry token, so we will probably need to mention that and provide a link to the setup stuff for NGC. I can help with this if needed.

File: startdash.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#!/opt/conda/envs/rapids/bin/python3
+#!/usr/bin/env python3
```
Comment (mlgill) [inline startdash.py:1]: Might need to edit the other python files this way as well? Can also do in a subsequent PR.

**Review by mlgill — APPROVED** [review]
Minor changes requested -- I've approved though.

File: startdash.py (hunk 4 above)
Comment (rilango) [reply startdash.py:1]: I think, this might be the only entry-point script.

**Pull request merged**